requirements.txt

```diff
@@ -34,9 +34,12 @@ pytz>=2019.2
 pyyaml>=3.13
 requests>=2.21.1
 scikit-learn>=0.20.2
+scipy>=1.4.1
 seaborn>=0.9.0
 setuptools>=40.6.3
+statsmodels>=0.11.1
```

Comment:
Can you add these the conda/conda-reqs.txt as well (otherwise you'll get a test failiure)

Reply:
Added check. test passing now.